main.tf

```diff
@@ -503,3 +503,24 @@ resource "aws_iam_role_policy_attachment" "eip" {
   role       = aws_iam_role.instance.name
   policy_arn = aws_iam_policy.eip[0].arn
 }
+
+################################################################################
+### Lambda function for ASG instance termination lifecycle hook
+################################################################################
+module "terminate_workers_lifecycle_function" {
+  source = "./modules/terminate-workers"
```

Comment:
question: Should we rename this to `terminate-runner-instances`? I checked the the Gitlab documentation and they have a Runner (the agent or coordinator running 24/7 and requesting new jobs) and Runner Instances (the machines doing the work). I am also not very precise with the wording in my PRs, but I think it would be better to do so. This should be done for all other occurrences as well.

Reply:
I replaced 'workers' with 'instances' in a290efa4934c4a43c3f14143be6f21d5ba1ef689